packages/app/src/cli/services/environment/identifiers.ts

```diff
@@ -12,6 +12,7 @@ export interface EnsureDeploymentIdsPresenceOptions {
   appId: string
   appName: string
   envIdentifiers: Partial<Identifiers>
+  force: boolean
```

Comment:
In this same file, should we remove the `environment.local.useFunctionMatching()` check and enable function matching always? Don't think it makes sense to keep it under a flag now that the deploy is not strict

Reply:
sure thing, thanks!